web_syncer/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://www.gnu.org/graphics/lgplv3-147x51.png
+   :target: https://www.gnu.org/licenses/lgpl-3.0.en.html
+   :alt: License: LGPL-v3
+
+===============
+   Web Syncer
+===============
+
+This module provides generic interface to receive create, update and delete model notifications on web client side.
+
+Usage
+=====
+
+To use this module functionality you need to:
+
+- Inherit ``web.syncer`` model at the backend side
+
+
+.. code-block:: python
+
+      class Task(models.Model):
+         _inherit = 'project.task'
+         _implements_syncer = True
+
+- Instantiate web.syncer and subscribe to the notifications for your model
+
+.. code-block:: javascript
+
+      const Syncer = require('web.syncer').Syncer;
```

Comment:
can't we achieve this with simple triggering proper `bus` events in the proper places? Do we really need all this code? :thinking: 

Could you improve this README to better describe which features are you providing?
/cc @Yajo

Reply:
I'm pretty sure you're right @simahawk

Reply:
Hi, let me explain .... We created this module as a part of the https://github.com/OCA/project-agile project to be used in module ``lira`` which is a complete fresh UI for agile project management. We needed a more generic approach then we had with ``bus``, so we wrote this wrapper around it. The idea behind it is that once configured for a model the system (base model) will listen for changes on all fields, but not just simple fields, it will listen for changes of related many2one, one2many, many2many as well and it will send automatic updates via ``bus``.  I hope this explanation helped a bit ...